### PR TITLE
210905 Nepal 2019 JW

### PR DESCRIPTION
**Pull request by jianingwwww**
- Nepal2019 has welevel1 and welevel2 (only basic and secondary education status), so welevel2 is dropped before running the do file. The same goes for helevel and melevel.
- manually adjusted to be able to merge branch main

### CODE REVIEW

File: MICS6_Master.do [hunk 1]
```diff
@@ -76,6 +76,9 @@ foreach name in $newMICS6countries {
 	do "${DO}/4_sexual_health"
 
 * Housekeeping for women data
+	if inlist("`name'","Nepal2019") {
+	drop welevel2
```
Comment (wengxyu1030) [inline MICS6_Master.do:80]: Could you clarify why dorp the variables here as we have the "keep" command at the very end that drop all the variables that are not needed?

File: MICS6_Master.do [hunk 2]
```diff
@@ -202,6 +205,10 @@ foreach name in $newMICS6countries {
 	mmerge hh1 hh2 ln using `wm'
 	drop _merge
 	
+	if inlist("`name'","Nepal2019") {
+	drop melevel2
```
Comment (wengxyu1030) [inline MICS6_Master.do:209]: Could you clarify why dorp the variables here as we have the "keep" command at the very end that drop all the variables that are not needed?

File: MICS6_Master.do [hunk 3]
```diff
@@ -245,7 +252,9 @@ foreach name in $newMICS6countries {
 	drop if _merge == 2
 	drop _merge
 	gen country_name = "`name'"
-	
+	if inlist("`name'","Nepal2019") {
+	drop helevel2
```
Comment (wengxyu1030) [inline MICS6_Master.do:256]: Could you clarify why dorp the variables here as we have the "keep" command at the very end that drop all the variables that are not needed?

**Review by wengxyu1030 — COMMENTED** [review]
Hi @jianingwwww ,

Thanks for your input! I have a small question about your code on the master do file. Much appreciated it if you could kindly clarify it.

**Comment by jianingwwww** [comment]
> Hi @jianingwwww ,
> 
> Thanks for your input! I have a small question about your code on the master do file. Much appreciated it if you could kindly clarify it.

Dear @wengxyu1030 ,
 
I dropped the m/h/welevel2 because these variables will interfere with the running of the code by reporting the error " m/h/welevel ambiguous abbreviation" (as I have explained above, there are m/h/welevel1 and m/h/welevel2 in Nepal2019 and what we need is m/h/welevel1 ). So they had to be excluded before generating other variables with m/h/welevel1 to avoid error. 
Hope this could explain!

Best,
Jianing

**Comment by wengxyu1030** [comment]
Hi @jianingwwww , thanks for the detailed explanation, that is very helpful. I am adding the comments in the code for future reference use. 

Regards, 
Aline

**Pull request merged**